nipype/interfaces/afni/preprocess.py

```diff
@@ -1844,14 +1924,26 @@ class ROIStats(AFNICommandBase):
     input_spec = ROIStatsInputSpec
     output_spec = ROIStatsOutputSpec
 
-    def aggregate_outputs(self, runtime=None, needed_outputs=None):
-        outputs = self._outputs()
-        output_filename = 'roi_stats.csv'
-        with open(output_filename, 'w') as f:
-            f.write(runtime.stdout)
+    def _format_arg(self, name, spec, value):
 
-        outputs.stats = os.path.abspath(output_filename)
-        return outputs
+        _stat_dict  =   {
+                            'mean': '-nzmean',
+                            'median': '-nzmedian',
+                            'mode': '-nzmode',
+                            'minmax': '-nzminmax',
+                            'sigma': '-nzsigma',
+                            'voxels': '-nzvoxels',
+                            'sum': '-nzsum',
+                            'summary': '-summary',
+                            'zerominmax': '-minmax',
+                            'zeromedian': '-median',
+                            'zerosigma': '-sigma',
+                            'zeromode': '-mode'
+                        }
```

Comment:
We (try to) conform to PEP8. Can you restyle this as follows (4 space indentation, no double spaces, closing brace matching entries)?

```Python
_stat_dict = {
    'mean': '-nzmean',
    ...
    }
```